lib/Stats.js

```diff
@@ -460,20 +471,27 @@ Stats.jsonToString = function jsonToString(obj, useColors) {
 		colors.bold(obj.publicPath);
 		newline();
 	}
+
 	if(obj.assets && obj.assets.length > 0) {
 		var t = [
-			["Asset", "Size", "Chunks", "", "Chunk Names"]
+			[
+				{value: "Asset", color: colors.bold},
+				{value: "Size", color: colors.bold},
+				{value: "Chunks", color: colors.bold},
+				{value: "", color: colors.bold},
+				{value: "Chunk Names", color: colors.bold}
+			]
 		];
 		obj.assets.forEach(function(asset) {
 			t.push([
-				asset.name,
-				formatSize(asset.size),
-				asset.chunks.join(", "),
-				asset.emitted ? "[emitted]" : "",
-				asset.chunkNames.join(", ")
+				{value: asset.name, color: getAssetColor(asset, colors.green)}, 
+				{value: formatSize(asset.size), color: getAssetColor(asset, colors.normal)}, 
+				{value: asset.chunks.join(", "), color: colors.bold}, 
+				{value: asset.emitted ? "[emitted]" : "", color: colors.green}, 
```

Comment:
done

I like how it looks too: 

<img width="1267" alt="screen shot 2016-11-30 at 10 31 17 pm" src="https://cloud.githubusercontent.com/assets/3408176/20781927/c426cafe-b74c-11e6-9072-db3150ec3e1d.png">